.env.example

```diff
@@ -4,14 +4,20 @@ REACT_APP_RELAYER_URL='http://localhost:3001/api/v2'
 REACT_APP_FEE_RECIPIENT='0x0000000000000000000000000000000000000000'
 REACT_APP_NOTIFICATIONS_LIMIT=20
 REACT_APP_START_BLOCK_LIMIT=100000
-REACT_APP_UI_UPDATE_CHECK_INTERVAL=5000 # 0 is disabled
-REACT_APP_UPDATE_ETHER_PRICE_INTERVAL=3600000 # 0 is disabled
+# 0 is disabled
+REACT_APP_UI_UPDATE_CHECK_INTERVAL=5000
+# 0 is disabled
+REACT_APP_UPDATE_ETHER_PRICE_INTERVAL=3600000
 REACT_APP_LOGGER_ID='0x-launch-kit-frontend'
 REACT_APP_ENABLE_NO_METAMASK_PROMPT='true'
 REACT_APP_STEP_MODAL_DONE_STATUS_VISIBILITY_TIME='3500'
 REACT_APP_COLLECTIBLES_SOURCE='opensea'
-REACT_APP_COLLECTIBLE_ADDRESS='0xf5b0a3efb8e8e4c201e2a935f110eaaf3ffecb8d' # Axie mainnet address
+# Axie mainnet address
+REACT_APP_COLLECTIBLE_ADDRESS='0xf5b0a3efb8e8e4c201e2a935f110eaaf3ffecb8d'
 REACT_APP_COLLECTIBLE_NAME='CryptoKitties'
 REACT_APP_COLLECTIBLE_DESCRIPTION='CryptoKitties is a state of the art blockchain-based game, enabling users to trade and sell their cards freely, with the same level of ownership as if they were real, tangible cards.'
 REACT_APP_ERC20_THEME_NAME = 'DARK_THEME'
 REACT_APP_ERC721_THEME_NAME = 'LIGHT_THEME'
+REACT_APP_DEFAULT_BASE_PATH='/erc721'
+REACT_APP_CACHE_CHECK_INTERVAL='60000'
+REACT_APP_GIT_COMMIT='ed77670d4a366cf7cdf93da449024024bac8d0a3'
```

Comment:
This is mostly for us, I don't think it makes sense to have it in the `.env.example` file.

Reply:
Removed